src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/IncrementalHash.net46.cs

```diff
@@ -116,7 +114,7 @@ public byte[] GetHashAndReset()
                 _hash.Initialize();
             }
 
-            _hash.TransformFinalBlock(s_empty, 0, 0);
+            _hash.TransformFinalBlock(Array.Empty<byte>(), 0, 0);
```

Comment:
https://github.com/dotnet/corefx/pull/5005#discussion_r47748321

> I'm sure it made sense at the time :). (Just like I feel like our internal build didn't allow Array.Empty; hence s_empty below)

Apparently I'm in a self-perpetuating feedback loop.